src/Servers/Kestrel/Core/src/CoreStrings.resx

```diff
@@ -719,4 +719,4 @@ For more information on configuring HTTPS see https://go.microsoft.com/fwlink/?l
   <data name="ConnectResponseCanNotHaveBody" xml:space="preserve">
     <value>Responses to 'CONNECT' requests with the success status code '{StatusCode}' cannot have a response body. Use the 'IHttpExtendedConnectFeature' to accept and write to the 'CONNECT' stream.</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```

Comment:
If I make another push, I'll clean up this newline - otherwise, it doesn't seem worth re-running CI.